en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/configurations/configure-logs-enforcer.md

```diff
@@ -0,0 +1,158 @@
+-   **additivity**: Allows to inherit all the appenders of the parent Logger if set as `true`.
+
+In summary, in following steps yu can specify a logger.
+
+1. Add the logger in the `log4j2.properties` or `log4j2-configmap.yaml` file, [based on your Choreo Connect deployment](#filepaths) to the list of loggers by referring the logger name:
+
+    ```properties
+    loggers = mgw-enforcer, io-swagger-v3, mgw-enforcer-interceptors, <your_logger_name>, 
+    ```
+
+2. Add the logger in the `log4j2.properties` or `log4j2-configmap.yaml` file, [based on your Choreo Connect deployment](#filepaths) as you need:
+
+    ```properties
+    logger.<your_logger_name>.name = <Component_name>
+    logger.<your_logger_name>.type = INFO
+    ```
+
+    For example:
+
+    ```properties
+    logger.enforcer.name = org.wso2.micro.gateway.enforcer
+    logger.enforcer.level = INFO
+    logger.enforcer.additivity = false
+    logger.enforcer.appenderRef.rolling.ref = ENFORCER_LOGFILE
+    logger.enforcer.appenderRef.console.ref = ENFORCER_CONSOLE
+    ```
+
+!!! Note
+    You can find more details on log4j2 appenders, loggers and it's attributes from the official documentation of [log4j2](#https://logging.apache.org/log4j/2.x/). 
+
+### Setting the log format
+
+#### Plain Text format
+
+By default the enforcer is enabled with the plain text formatted logs. The plain text format is enabled with the `log4j2`'s `PatternLayout`. Following configuration under appender will specify that.
+
+```properties
+appender.appender_name.layout.type = PatternLayout
+```
+
+Then the pattern of the text formatted log will be specified by the following config.
+
+```properties
+appender.<appender_name>.layout.pattern = [%d{DEFAULT}][%X{traceId}]%x %5p - {\%c} - %m [%errorDetails]%n
+```
+
+!!! Note
+    More details on this pattern specification can be find in the official [log4j2 documentation](https://logging.apache.org/log4j/2.x/manual/layouts.html#PatternLayout).
+
+#### JSON format
+
+Enforcer can be configured to output logs in JSON format by setting the layout as `CustomJsonLayout` which is a customized imlementation of log4j2's [AbstractStringLayout](https://logging.apache.org/log4j/2.x/log4j-core/apidocs/org/apache/logging/log4j/core/layout/AbstractStringLayout.html). You can set it using the following configuration.
+
+
+```properties
+appender.<appender_name>.layout.type = CustomJsonLayout
+```
+
+!!! Note
+    Following configuration is not required here, as the pattern of a `CustomJsonLayout` layout cannot be specified.
+    ```properties
+    appender.<appender_name>.layout.pattern = <pattern_string>
+    ```
```

Comment:
need to revisit this

Reply:
Fixed in https://github.com/wso2/docs-apim/pull/5357/commits/4681da56484b9296768a350aab44df5936c04cb7